include/sway/config.h

```diff
@@ -40,6 +40,7 @@ enum binding_flags {
 	BINDING_TITLEBAR=16, // mouse only; trigger on container titlebar
 	BINDING_CODE=32,     // keyboard only; convert keysyms into keycodes
 	BINDING_RELOAD=62,   // switch only; (re)trigger binding on reload
```

Comment:
OT: Shouldn't this be 64?

Reply:
Yep.

Reply:
Separate PR or drive-by fix in this one? Have no idea how to and no real inclination of going down the rabbithole of retesting it, TBH. :-/ Had a quick glance where it's used and seems trivial enough, though.

Reply:
A second commit in this PR would be fine imo.

Reply:
Done. Thanks!